packages/asc-ui/src/components/Icon/IconStyle.ts

```diff
@@ -43,7 +41,11 @@ const IconStyle = styled.span<Props>`
     `}
   }
 
-  ${({ color, theme }) => svgFill(color)({ theme })};
+  ${({ color }) => css`
```

Comment:
Let's use the `svgFill` util here

Reply:
@chrisvanmook `svgFill` should also be refactored actually so that it takes a color directly, or is there an easy fix here?

Reply:
Yes it needs to be refactored. Out of scope for this ticket, I'll create an issue. @jasperswart could you make the css like so:
```
& svg {
            circle,
            rect,
            polygon,
            path {
              fill: ${color};
            }
}
```

Reply:
the svgFill is used 46 times in a lot of differtent components. if we break svgFill then those will break as well.

Reply:
or maybe add: svgFillColor util for this variant.

Reply:
@chrisvanmook I have changed the css as you asked

Reply:
Yeah, let's use the example CSS that @chrisvanmook showed. We can use the other ticket to refactor it.

Reply:
Looks good!